### PR TITLE
zc 0.14.0 (new cask)

### PR DESCRIPTION
**Pull request by zectdev**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --cask --new <cask>` worked successfully.
- [X] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by daeho-ro** [comment]
It seems that the binary should go to the homebrew core first.

**Comment by bevanjkay** [comment]
If the binary is built from a closed-source then `homebrew-cask` is the appropriate place for this to be submitted.

Aside to this, I am not convinced that the app carries enough notability to be included at this stage. Self-submission (by the vendor) is often an initial sign that there is not demand for the cask to be included in `homebrew-cask`. Additionally, a lack of SSL certificate on the homepage and download links points to the project not having a required level of maturity to be included as is.

It may be better to begin by offering this in  your [own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).

**Comment by zectdev** [comment]
This is indeed built closed-source.  We use a wildcard SSL cert https://releases.zectonal.com/ and https://www.zectonal.com/

**Comment by bevanjkay** [comment]
Why not use those links in the cask file?

**Comment by zectdev** [comment]
I didn't think of it since this is the first attempt, but I could easily do that...what is the process to handle this PR if I go back and use an HTTPS URL?  Should I resubmit a new PR?

**Comment by bevanjkay** [comment]
You can push additional commits to this PR. We don't currently require commits to be squashed in `homebrew-cask` either.

**Comment by zectdev** [comment]
Great news...I will update and push the new commits to this PR - thank you for the feedback.

**Comment by zectdev** [comment]
I updated with https that uses our SSL cert

File: Casks/z/zc.rb [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
+  url "https://docs.zectonal.com/zc/releases/download/v#{version}/zc_#{version}_macos.zip"
   name "zc"
   desc "Blazingly fast data observability"
-  homepage "http://zectonal-releases.s3-website-us-east-1.amazonaws.com/index.html"
+  homepage "https://docs.zectonal.com/index.html"
 
   livecheck do
-    url "http://zectonal-releases.s3-website-us-east-1.amazonaws.com/zc/releases/download/index.html"
+    url "https://docs.zectonal.com/zc/releases/download/index.html"
     strategy :page_match
```
Comment (bevanjkay) [inline Casks/z/zc.rb:12]:
```suggestion
```
The `strategy` can be removed as `page_match` is the default.

File: Casks/z/zc.rb [hunk 2]
```diff
@@ -2,16 +2,15 @@
...
-    # regex(%r{href=.*?\"v(\d+(?:\.\d+)*)\/index\.html\"}i)
-    regex(%r{href=.*?"http://zectonal-releases\.s3-website-us-east-1\.amazonaws\.com/zc/releases/download/v(\d+(?:\.\d+)*).*}i)
+    regex(/href=.*v(\d+(?:\.\d+)*).*/i)
```
Comment (bevanjkay) [inline Casks/z/zc.rb:13]:
```suggestion
    regex(%r{href=.*?download/v?(\d+(?:\.\d+)+)/index.html}i)
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/z/zc.rb (hunk 2 above)
Comment (zectdev) [reply Casks/z/zc.rb]: thank you for this feedback.  I incorporated into the latest commit below.

**Comment by razvanazamfirei** [comment]
If no zap is needed for this cask, we should add `  # No zap stanza required`. 

Regardless, I agree with @bevanjkay that this does not meet our notability [requirements](https://docs.brew.sh/Acceptable-Casks#rejected-casks). 


Thank you for the PR, @zectdev and we really appreciate your contribution! Unfortunately, we're going to pass on this for now.

**Pull request closed without merging**